tests/test_sklearn_decision_tree_converters.py

```diff
@@ -33,56 +33,63 @@ def test_random_forest_implementation(self):
                     str(type(list(pytorch_model.operator_map.values())[0])) == dt_implementation_map[extra_config_param]
                 )
 
-    def _run_random_forest_classifier_converter(self, num_classes, extra_config={}, labels_shift=0):
+    # Used for classification tests
+    def _run_tree_classification_converter(self, model_type, num_classes, extra_config={}, labels_shift=0, **kwargs):
         warnings.filterwarnings("ignore")
         for max_depth in [1, 3, 8, 10, 12, None]:
```

Comment:
For my info: On what basis did we originally "tune"/identify these depth values?

Reply:
We use these values based on the pipelines we used in the experiments in the paper.

Reply:
I see. Will read it and reach out if there's anything. Thanks!